### DISCUSSION TITLE
Yarn install is throwing an error

### DISCUSSION THREAD

**Discussion by akash1412** [adobe/react-spectrum#4351] [General]
Hi, I'm trying to set up the project on my Windows system, but it throws an error while executing the command `yarn install` throws an error. 
I don't know if I'm missing something here.
`yarn version:1.22.19`

[4/4] Building fresh packages...
$ yarn build:icons && patch-package
yarn run v1.22.19
$ babel-node --presets @babel/env ./scripts/buildIcons.js
$ babel-node --extensions '.cjs,.js' --presets @babel/env ./scripts/generateIcons.cjs -i '[]' && yarn generate-types
$ babel-node --extensions '.cjs,.js' --presets @babel/env ./scripts/generateIcons.cjs -i '[]' && yarn generate-types
$ babel-node --extensions '.cjs,.js' --presets @babel/env ./scripts/generateIcons.cjs -i '[]' && yarn generate-types
$ babel-node --extensions '.cjs,.js' --presets @babel/env ./scripts/generateIcons.cjs -i '[]' && yarn generate-types
[Making and building icons for @spectrum-icons/workflow] (node:15120) Warning: To load an ES module, set "type": "module" in the package.json or use the .mjs extension.
[Making and building icons for @spectrum-icons/workflow] (Use `node --trace-warnings ...` to show where the warning was created)
[Making and building icons for @spectrum-icons/workflow] F:\react-spectrum\packages\@spectrum-icons\workflow\scripts\generateIcons.cjs:13
[Making and building icons for @spectrum-icons/workflow] import {generateIcons} from '@spectrum-icons/build-tools/generateIcons';
[Making and building icons for @spectrum-icons/workflow] ^^^^^^
[Making and building icons for @spectrum-icons/workflow]
[Making and building icons for @spectrum-icons/workflow] SyntaxError: Cannot use import statement outside a module
[Making and building icons for @spectrum-icons/workflow]     at Object.compileFunction (node:vm:352:18)
[Making and building icons for @spectrum-icons/workflow]     at wrapSafe (node:internal/modules/cjs/loader:1032:15)
[Making and building icons for @spectrum-icons/workflow]     at Module._compile (node:internal/modules/cjs/loader:1067:27)
[Making and building icons for @spectrum-icons/workflow]     at Object.Module._extensions..js (node:internal/modules/cjs/loader:1155:10)
[Making and building icons for @spectrum-icons/workflow]     at Module.load (node:internal/modules/cjs/loader:981:32)
[Making and building icons for @spectrum-icons/workflow]     at Function.Module._load (node:internal/modules/cjs/loader:822:12)
[Making and building icons for @spectrum-icons/workflow]     at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12)
[Making and building icons for @spectrum-icons/workflow]     at Object.<anonymous> (F:\react-spectrum\node_modules\@babel\node\lib\_babel-node.js:172:21)
[Making and building icons for @spectrum-icons/workflow]     at Module._compile (node:internal/modules/cjs/loader:1103:14)
[Making and building icons for @spectrum-icons/workflow]     at Object.Module._extensions..js (node:internal/modules/cjs/loader:1155:10)
[Making and building icons for @spectrum-icons/ui] (node:7812) Warning: To load an ES module, set "type": "module" in the package.json or use the .mjs extension.
[Making and building icons for @spectrum-icons/ui] (Use `node --trace-warnings ...` to show where the warning was created)
[Making and building icons for @spectrum-icons/ui] F:\react-spectrum\packages\@spectrum-icons\ui\scripts\generateIcons.cjs:13
[Making and building icons for @spectrum-icons/ui] import {compileSVG} from '@spectrum-icons/build-tools/compileSVG';
[Making and building icons for @spectrum-icons/ui] ^^^^^^
[Making and building icons for @spectrum-icons/ui]
[Making and building icons for @spectrum-icons/ui] SyntaxError: Cannot use import statement outside a module
[Making and building icons for @spectrum-icons/ui]     at Object.compileFunction (node:vm:352:18)
[Making and building icons for @spectrum-icons/ui]     at wrapSafe (node:internal/modules/cjs/loader:1032:15)
[Making and building icons for @spectrum-icons/ui]     at Module._compile (node:internal/modules/cjs/loader:1067:27)
[Making and building icons for @spectrum-icons/ui]     at Object.Module._extensions..js (node:internal/modules/cjs/loader:1155:10)
[Making and building icons for @spectrum-icons/ui]     at Module.load (node:internal/modules/cjs/loader:981:32)
[Making and building icons for @spectrum-icons/ui]     at Function.Module._load (node:internal/modules/cjs/loader:822:12)
[Making and building icons for @spectrum-icons/ui]     at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12)
[Making and building icons for @spectrum-icons/ui]     at Object.<anonymous> (F:\react-spectrum\node_modules\@babel\node\lib\_babel-node.js:172:21)
[Making and building icons for @spectrum-icons/ui]     at Module._compile (node:internal/modules/cjs/loader:1103:14)
[Making and building icons for @spectrum-icons/ui]     at Object.Module._extensions..js (node:internal/modules/cjs/loader:1155:10)
[Making and building icons for @spectrum-icons/express] (node:6844) Warning: To load an ES module, set "type": "module" in the package.json or use the .mjs extension.
[Making and building icons for @spectrum-icons/express] (Use `node --trace-warnings ...` to show where the warning was created)
[Making and building icons for @spectrum-icons/color] (node:6908) Warning: To load an ES module, set "type": "module" in the package.json or use the .mjs extension.
[Making and building icons for @spectrum-icons/color] (Use `node --trace-warnings ...` to show where the warning was created)
[Making and building icons for @spectrum-icons/color] F:\react-spectrum\packages\@spectrum-icons\color\scripts\generateIcons.cjs:13
[Making and building icons for @spectrum-icons/color] import {generateIcons} from '@spectrum-icons/build-tools/generateIcons';
[Making and building icons for @spectrum-icons/color] ^^^^^^
[Making and building icons for @spectrum-icons/color]
[Making and building icons for @spectrum-icons/color] SyntaxError: Cannot use import statement outside a module
[Making and building icons for @spectrum-icons/color]     at Object.compileFunction (node:vm:352:18)
[Making and building icons for @spectrum-icons/color]     at wrapSafe (node:internal/modules/cjs/loader:1032:15)
[Making and building icons for @spectrum-icons/color]     at Module._compile (node:internal/modules/cjs/loader:1067:27)
[Making and building icons for @spectrum-icons/color]     at Object.Module._extensions..js (node:internal/modules/cjs/loader:1155:10)
[Making and building icons for @spectrum-icons/color]     at Module.load (node:internal/modules/cjs/loader:981:32)
[Making and building icons for @spectrum-icons/color]     at Function.Module._load (node:internal/modules/cjs/loader:822:12)
[Making and building icons for @spectrum-icons/color]     at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12)
[Making and building icons for @spectrum-icons/color]     at Object.<anonymous> (F:\react-spectrum\node_modules\@babel\node\lib\_babel-node.js:172:21)
[Making and building icons for @spectrum-icons/color]     at Module._compile (node:internal/modules/cjs/loader:1103:14)
[Making and building icons for @spectrum-icons/color]     at Object.Module._extensions..js (node:internal/modules/cjs/loader:1155:10)
[Making and building icons for @spectrum-icons/express] F:\react-spectrum\packages\@spectrum-icons\express\scripts\generateIcons.cjs:13
[Making and building icons for @spectrum-icons/express] import fs from 'fs-extra';
[Making and building icons for @spectrum-icons/express] ^^^^^^
[Making and building icons for @spectrum-icons/express]
[Making and building icons for @spectrum-icons/express] SyntaxError: Cannot use import statement outside a module
[Making and building icons for @spectrum-icons/express]     at Object.compileFunction (node:vm:352:18)
[Making and building icons for @spectrum-icons/express]     at wrapSafe (node:internal/modules/cjs/loader:1032:15)
[Making and building icons for @spectrum-icons/express]     at Module._compile (node:internal/modules/cjs/loader:1067:27)
[Making and building icons for @spectrum-icons/express]     at Object.Module._extensions..js (node:internal/modules/cjs/loader:1155:10)
[Making and building icons for @spectrum-icons/express]     at Module.load (node:internal/modules/cjs/loader:981:32)
[Making and building icons for @spectrum-icons/express]     at Function.Module._load (node:internal/modules/cjs/loader:822:12)
[Making and building icons for @spectrum-icons/express]     at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12)
[Making and building icons for @spectrum-icons/express]     at Object.<anonymous> (F:\react-spectrum\node_modules\@babel\node\lib\_babel-node.js:172:21)
[Making and building icons for @spectrum-icons/express]     at Module._compile (node:internal/modules/cjs/loader:1103:14)
[Making and building icons for @spectrum-icons/express]     at Object.Module._extensions..js (node:internal/modules/cjs/loader:1155:10)
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.um-icons/workflow]
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.um-icons/ui]
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.um-icons/express]
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
[Making and building icons for @spectrum-icons/ui] yarn make-icons && yarn build-icons exited with code 1
[Making and building icons for @spectrum-icons/express] yarn make-icons && yarn build-icons exited with code 1
[Making and building icons for @spectrum-icons/color] yarn make-icons && yarn build-icons exited with code 1
[Making and building icons for @spectrum-icons/workflow] yarn make-icons && yarn build-icons exited with code 1
node:internal/process/promises:265
            triggerUncaughtException(err, true /* fromPromise */);
            ^

[UnhandledPromiseRejection: This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason "[object Array]".] {
  code: 'ERR_UNHANDLED_REJECTION'
}
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.

**Comment by snowystinger** [comment]
Sorry to hear that, what's your node version?

**Reply by akash1412** [reply]
Using `v16.14.0`

**Reply by archie252000** [reply]
did you resolve it?

**Reply by snowystinger** [reply]
I'm sorry, this fell off our radar. that node version should have worked. We're on 16.20 now if you want to try that. Otherwise, we'll need you to do some more digging, what line in particular is throwing this error that isn't caught?

**Reply by archie252000** [reply]
The primary issue from the error seems to be the import error. " Cannot use import statement outside a module" in the @spectrum-icons.

**Reply by snowystinger** [reply]
for some reason it looks like babel-node isn't getting used, node shouldn't be seeing those as import statements

I don't see babel-cli (which contains babel-node) as an explicit dependency in our repo, could you see if you have it installed?

**Reply by archie252000** [reply]
babel is present:
![image](https://github.com/adobe/react-spectrum/assets/51921550/303f35f8-4593-4999-aa07-3d288b137c1a)

**Reply by archie252000** [reply]
Here is the full console output:
yarn install
yarn install v1.22.19
[1/4] Resolving packages...
warning Resolution field "@babel/core@7.12.10" is incompatible with requested version "@babel/core@^7.15.5"
warning Resolution field "browserslist@4.20.3" is incompatible with requested version "browserslist@^1.7.6"
warning Resolution field "react-refresh@0.9.0" is incompatible with requested version "react-refresh@^0.11.0"
warning Resolution field "@babel/core@7.12.10" is incompatible with requested version "@babel/core@7.12.9"
warning Resolution field "@babel/core@7.12.10" is incompatible with requested version "@babel/core@7.12.9"
[2/4] Fetching packages...
[3/4] Linking dependencies...
warning "@babel/preset-env > @babel/plugin-bugfix-v8-spread-parameters-in-optional-chaining@7.13.12" has incorrect peer dependency "@babel/core@^7.13.0".
warning "@storybook/addon-actions > react-inspector@5.1.1" has incorrect peer dependency "react@^16.8.4 || ^17.0.0".
warning " > @storybook/react@6.5.12" has unmet peer dependency "require-from-string@^2.0.2".
warning "@storybook/react > @pmmmwh/react-refresh-webpack-plugin@0.5.6" has incorrect peer dependency "react-refresh@>=0.10.0 <1.0.0".
warning "@storybook/react > react-element-to-jsx-string@14.3.4" has incorrect peer dependency "react@^0.14.8 || ^15.0.1 || ^16.0.0 || ^17.0.1".
warning "@storybook/react > react-element-to-jsx-string@14.3.4" has incorrect peer dependency "react-dom@^0.14.8 || ^15.0.1 || ^16.0.0 || ^17.0.1".
warning " > @storybook/testing-react@1.3.0" has unmet peer dependency "@storybook/client-api@>=6.4.0".
warning " > @storybook/testing-react@1.3.0" has unmet peer dependency "@storybook/preview-web@>=6.4.0".
warning " > react-test-renderer@16.10.2" has incorrect peer dependency "react@^16.0.0".
warning " > storybook-dark-mode@1.1.1-canary.120.3843.0" has unmet peer dependency "@storybook/core-events@^6.0.0".
warning " > storybook-dark-mode@1.1.1-canary.120.3843.0" has unmet peer dependency "@storybook/theming@^6.0.0".
warning "verdaccio > clipanion@3.1.0" has unmet peer dependency "typanion@*".
warning "workspace-aggregator-327cd593-df43-47dc-9e4f-b9e86b1bdb3c > @react-spectrum/docs > quicklink@2.0.0" has incorrect peer dependency "react@^16.8.0".
warning "workspace-aggregator-327cd593-df43-47dc-9e4f-b9e86b1bdb3c > @react-spectrum/docs > quicklink@2.0.0" has incorrect peer dependency "react-dom@^16.8.0".
warning Workspaces can only be enabled in private projects.
warning Workspaces can only be enabled in private projects.
warning Workspaces can only be enabled in private projects.
[4/4] Building fresh packages...
$ yarn build:icons && patch-package
yarn run v1.22.19
$ babel-node --presets @babel/env ./scripts/buildIcons.js
$ babel-node --extensions '.cjs,.js' --presets @babel/env ./scripts/generateIcons.cjs -i '[]' && yarn generate-types
$ babel-node --extensions '.cjs,.js' --presets @babel/env ./scripts/generateIcons.cjs -i '[]' && yarn generate-types
$ babel-node --extensions '.cjs,.js' --presets @babel/env ./scripts/generateIcons.cjs -i '[]' && yarn generate-types
$ yarn build-cjs && yarn build-esm && yarn generate-types
$ babel-node --extensions '.cjs,.js' --presets @babel/env ./scripts/generateIcons.cjs -i '[]' && yarn generate-types
$ cross-env BUILD_ENV=production babel --root-mode upward src -d . --extensions '.ts,.tsx'
[Making and building icons for @spectrum-icons/workflow] (node:14248) Warning: To load an ES module, set "type": "module" in the package.json or use the .mjs extension.
[Making and building icons for @spectrum-icons/workflow] (Use `node --trace-warnings ...` to show where the warning was created)
[Making and building icons for @spectrum-icons/workflow] D:\Open Source\Adobe\react-spectrum\packages\@spectrum-icons\workflow\scripts\generateIcons.cjs:13
[Making and building icons for @spectrum-icons/workflow] import {generateIcons} from '@spectrum-icons/build-tools/generateIcons';
[Making and building icons for @spectrum-icons/workflow] ^^^^^^
[Making and building icons for @spectrum-icons/workflow]
[Making and building icons for @spectrum-icons/workflow] SyntaxError: Cannot use import statement outside a module
[Making and building icons for @spectrum-icons/workflow]     at internalCompileFunction (node:internal/vm:73:18)
[Making and building icons for @spectrum-icons/workflow]     at wrapSafe (node:internal/modules/cjs/loader:1176:20)
[Making and building icons for @spectrum-icons/workflow]     at Module._compile (node:internal/modules/cjs/loader:1218:27)
[Making and building icons for @spectrum-icons/workflow]     at Module._extensions..js (node:internal/modules/cjs/loader:1308:10)
[Making and building icons for @spectrum-icons/workflow]     at Module.load (node:internal/modules/cjs/loader:1117:32)
[Making and building icons for @spectrum-icons/workflow]     at Module._load (node:internal/modules/cjs/loader:958:12)
[Making and building icons for @spectrum-icons/workflow]     at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)
[Making and building icons for @spectrum-icons/workflow]     at Object.<anonymous> (D:\Open Source\Adobe\react-spectrum\node_modules\@babel\node\lib\_babel-node.js:172:21)
[Making and building icons for @spectrum-icons/workflow]     at Module._compile (node:internal/modules/cjs/loader:1254:14)
[Making and building icons for @spectrum-icons/workflow]     at Module._extensions..js (node:internal/modules/cjs/loader:1308:10)
[Making and building icons for @spectrum-icons/workflow]
[Making and building icons for @spectrum-icons/workflow] Node.js v18.16.0
[Making and building icons for @spectrum-icons/ui] (node:18584) Warning: To load an ES module, set "type": "module" in the package.json or use the .mjs extension.
[Making and building icons for @spectrum-icons/ui] (Use `node --trace-warnings ...` to show where the warning was created)
[Making and building icons for @spectrum-icons/ui] D:\Open Source\Adobe\react-spectrum\packages\@spectrum-icons\ui\scripts\generateIcons.cjs:13
[Making and building icons for @spectrum-icons/ui] import {compileSVG} from '@spectrum-icons/build-tools/compileSVG';
[Making and building icons for @spectrum-icons/ui] ^^^^^^
[Making and building icons for @spectrum-icons/ui]
[Making and building icons for @spectrum-icons/ui] SyntaxError: Cannot use import statement outside a module
[Making and building icons for @spectrum-icons/ui]     at internalCompileFunction (node:internal/vm:73:18)
[Making and building icons for @spectrum-icons/ui]     at wrapSafe (node:internal/modules/cjs/loader:1176:20)
[Making and building icons for @spectrum-icons/ui]     at Module._compile (node:internal/modules/cjs/loader:1218:27)
[Making and building icons for @spectrum-icons/ui]     at Module._extensions..js (node:internal/modules/cjs/loader:1308:10)
[Making and building icons for @spectrum-icons/ui]     at Module.load (node:internal/modules/cjs/loader:1117:32)
[Making and building icons for @spectrum-icons/ui]     at Module._load (node:internal/modules/cjs/loader:958:12)
[Making and building icons for @spectrum-icons/ui]     at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)
[Making and building icons for @spectrum-icons/ui]     at Object.<anonymous> (D:\Open Source\Adobe\react-spectrum\node_modules\@babel\node\lib\_babel-node.js:172:21)
[Making and building icons for @spectrum-icons/ui]     at Module._compile (node:internal/modules/cjs/loader:1254:14)
[Making and building icons for @spectrum-icons/ui]     at Module._extensions..js (node:internal/modules/cjs/loader:1308:10)
[Making and building icons for @spectrum-icons/ui]
[Making and building icons for @spectrum-icons/ui] Node.js v18.16.0
[Making and building icons for @spectrum-icons/color] (node:12728) Warning: To load an ES module, set "type": "module" in the package.json or use the .mjs extension.
[Making and building icons for @spectrum-icons/color] (Use `node --trace-warnings ...` to show where the warning was created)
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.um-icons/workflow]
[Making and building icons for @spectrum-icons/express] (node:20536) Warning: To load an ES module, set "type": "module" in the package.json or use the .mjs extension.
[Making and building icons for @spectrum-icons/express] (Use `node --trace-warnings ...` to show where the warning was created)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
[Making and building icons for @spectrum-icons/workflow] yarn make-icons && yarn build-icons exited with code 1
[Making and building icons for @spectrum-icons/ui] yarn make-icons && yarn build-icons exited with code 1
[Making and building icons for @spectrum-icons/express] D:\Open Source\Adobe\react-spectrum\packages\@spectrum-icons\express\scripts\generateIcons.cjs:13
[Making and building icons for @spectrum-icons/express] import fs from 'fs-extra';
[Making and building icons for @spectrum-icons/express] ^^^^^^
[Making and building icons for @spectrum-icons/express]
[Making and building icons for @spectrum-icons/express] SyntaxError: Cannot use import statement outside a module
[Making and building icons for @spectrum-icons/express]     at internalCompileFunction (node:internal/vm:73:18)
[Making and building icons for @spectrum-icons/express]     at wrapSafe (node:internal/modules/cjs/loader:1176:20)
[Making and building icons for @spectrum-icons/express]     at Module._compile (node:internal/modules/cjs/loader:1218:27)
[Making and building icons for @spectrum-icons/express]     at Module._extensions..js (node:internal/modules/cjs/loader:1308:10)
[Making and building icons for @spectrum-icons/express]     at Module.load (node:internal/modules/cjs/loader:1117:32)
[Making and building icons for @spectrum-icons/express]     at Module._load (node:internal/modules/cjs/loader:958:12)
[Making and building icons for @spectrum-icons/express]     at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)
[Making and building icons for @spectrum-icons/express]     at Object.<anonymous> (D:\Open Source\Adobe\react-spectrum\node_modules\@babel\node\lib\_babel-node.js:172:21)
[Making and building icons for @spectrum-icons/express]     at Module._compile (node:internal/modules/cjs/loader:1254:14)
[Making and building icons for @spectrum-icons/express]     at Module._extensions..js (node:internal/modules/cjs/loader:1308:10)
[Making and building icons for @spectrum-icons/express]
[Making and building icons for @spectrum-icons/express] Node.js v18.16.0
[Making and building icons for @spectrum-icons/color] D:\Open Source\Adobe\react-spectrum\packages\@spectrum-icons\color\scripts\generateIcons.cjs:13
[Making and building icons for @spectrum-icons/color] import {generateIcons} from '@spectrum-icons/build-tools/generateIcons';
[Making and building icons for @spectrum-icons/color] ^^^^^^
[Making and building icons for @spectrum-icons/color]
[Making and building icons for @spectrum-icons/color] SyntaxError: Cannot use import statement outside a module
[Making and building icons for @spectrum-icons/color]     at internalCompileFunction (node:internal/vm:73:18)
[Making and building icons for @spectrum-icons/color]     at wrapSafe (node:internal/modules/cjs/loader:1176:20)
[Making and building icons for @spectrum-icons/color]     at Module._compile (node:internal/modules/cjs/loader:1218:27)
[Making and building icons for @spectrum-icons/color]     at Module._extensions..js (node:internal/modules/cjs/loader:1308:10)
[Making and building icons for @spectrum-icons/color]     at Module.load (node:internal/modules/cjs/loader:1117:32)
[Making and building icons for @spectrum-icons/color]     at Module._load (node:internal/modules/cjs/loader:958:12)
[Making and building icons for @spectrum-icons/color]     at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)
[Making and building icons for @spectrum-icons/color]     at Object.<anonymous> (D:\Open Source\Adobe\react-spectrum\node_modules\@babel\node\lib\_babel-node.js:172:21)
[Making and building icons for @spectrum-icons/color]     at Module._compile (node:internal/modules/cjs/loader:1254:14)
[Making and building icons for @spectrum-icons/color]     at Module._extensions..js (node:internal/modules/cjs/loader:1308:10)
[Making and building icons for @spectrum-icons/color]
[Making and building icons for @spectrum-icons/color] Node.js v18.16.0
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.um-icons/express]
[Making and building icons for @spectrum-icons/express] yarn make-icons && yarn build-icons exited with code 1
[Making and building icons for @spectrum-icons/color] yarn make-icons && yarn build-icons exited with code 1
[Building icons for @spectrum-icons/illustrations] Successfully compiled 9 files with Babel (1202ms).
$ cross-env BUILD_ENV=production babel src -d . --extensions '.ts,.tsx' --out-file-extension '.module.mjs' --config-file '../../../babel-esm.config.json'
[Building icons for @spectrum-icons/illustrations] Successfully compiled 9 files with Babel (791ms).
$ tsc --project ./tsconfig.types.json
[Building icons for @spectrum-icons/illustrations] yarn build-icons exited with code 0
node:internal/process/promises:288
            triggerUncaughtException(err, true /* fromPromise */);
            ^

[UnhandledPromiseRejection: This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason "[object Array]".] {
  code: 'ERR_UNHANDLED_REJECTION'
}

Node.js v18.16.0
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.

**Reply by archie252000** [reply]
Working now. I don't know for some reason it does not work on Windows, using wsl it worked.
